desktop/renderer-app/src/react-app-env.d.ts

```diff
@@ -1,66 +0,0 @@
-/// <reference types="node" />
```

Comment:
It looks like this file should not have been removed, and now the compilation fails because of his removal.

This is strange because this file was created by create-react-app as I recall, and this project has removed create-react-app, but I didn't expect this file to still be functional

Anyway, let's restore this file first

CC: @crimx @hyrious @Cheerego7 do you know why?

Reply:
The `.d.ts` file works regardless of the file name. It was created by the create-react-app but looks like we also added other type declarations in it.

We rely on the `NodeJS` namespace so the `/// <reference types="node" />` cannot be removed.

Reply:
makes sense!